manuscript/draft.Rmd

```diff
@@ -81,53 +83,57 @@ in the future.
     rather than averaged over the full multi-decade period (as Worldclim does)
   * NDVI (GIMMS)
   * Altitude (SRTM 90m Digital Elevation Database via raster::getData)
-* Data summary: 
+* Data summary:
   * Number of sites, species, unique observers
   * central moments of richness (mean about 51, sd about 12 spp)
   * The within-site variation was much smaller (sd about 5 spp), making the
     site-level distributions underdispersed.  
     * This underdispersion implies
-      that common count models such as binomial, Poisson, negative binomial 
+      that common count models such as binomial, Poisson, negative binomial
       won't fit well.
-    * For this reason (and because the time series methods assumed 
-      Gaussian errors), we defined model performance throughout this paper in 
+    * For this reason (and because the time series methods assumed
+      Gaussian errors), we defined model performance throughout this paper in
       terms of Gaussians (e.g. mean squared error), rather than with count
       distributions.
         * Some of the modeling approaches (SDMs and JSDMs) did use discrete
           distributions internally, however.
 * Splitting the data temporally
-  * Our data set began in `r settings$start_yr`, which was the first year that 
+  * Our data set began in `r settings$start_yr`, which was the first year that
     NDVI data was available from GIMMS
-  * All the models were trained on the data that was collected through 
+  * All the models were trained on the data that was collected through
     `r settings$last_train_year`
-  * Sites were included in the analysis if there were at least 
+  * Sites were included in the analysis if there were at least
     `r settings$min_num_yrs` years of data available during this period.
-  * The models were tested/evaluated on the data from 
-    `r settings$last_train_year + 1` `r settings$end_yr`.
+  * The models were tested/evaluated on the data from
+    `r settings$last_train_year + 1` `r settings$end_yr`.  
+
+  Bird species richness was compiled from the Breeding Bird Survey (BBS) obtained using the Data Retriever software [@pardieck2017, @morris203]. The BBS data was initially filtered to exclude all aquatic species, unidentified species, and hybrids. All data from routes that did not meet BBS acceptable criteria were excluded. Only routes within the continental U.S.A were used so as to match the climatic dataset.
```

Comment:
"aquatic species..." -> "nocturnal, crepuscular, and aquatic species (since these species are not well sampled by BBS methods), as well as unidentified...".